pkgs/tools/inputmethods/libinput-gestures/default.nix

```diff
@@ -0,0 +1,35 @@
+{lib, stdenv, fetchFromGitHub, bash, fakeroot, libinput, xdotool, makeWrapper}:
+stdenv.mkDerivation rec {
+  pname = "libinput-gestures";
+  version = "2.32";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "bulletmark";
+    repo = "libinput-gestures";
+    rev = version;
+    sha256 = "1by6sabx0s8sd9w5675gc26q7yccxnxxsjg4dqlb6nbs0vcg81s7";
+  };
+
+  buildInputs = [ bash fakeroot makeWrapper ];
+
+  installPhase =
+    ''
+      mkdir -p $out
+      fakeroot -- bash libinput-gestures-setup -d "$out" install
+      mv $out/usr/bin $out/bin-unwrapped
+      mv $out/usr/share $out/share
+      rmdir $out/usr
+      mkdir $out/bin
+      sed -i "s,/usr/bin/,,g" $out/share/applications/*.desktop
+      makeWrapper $out/bin-unwrapped/libinput-gestures $out/bin/libinput-gestures --prefix PATH : ${lib.makeBinPath [ libinput xdotool ]}
+      makeWrapper $out/bin-unwrapped/libinput-gestures-setup $out/bin/libinput-gestures-setup --prefix PATH : ${lib.makeBinPath [  ]}
```

Comment:
What is the point of prefixing with an empty list?

Reply:
None, I forgot to remove this after finding out that -setup is pretty much useless in NixOS anyway.